analysis_runner/cromwell.py

```diff
@@ -386,6 +388,12 @@ def _get_wait_interval(
                 continue
             status = r.json().get('status')
             _remaining_exceptions = max_sequential_exception_count
+
+            # if workflow has concluded print logging to hail batch log
+            if status.lower() in terminal_statuses:
+                logger.info('Cromwell workflow has concluded - fetching log')
+                _check_cromwell_status(workflow_id, json_output='')
```

Comment:
Or feel free to change the definition to default to None
```suggestion
                _check_cromwell_status(workflow_id, json_output=None)
```

Reply:
I didn't spot this before - the type hinting is Optional but it has no default. I've set the default to `None` in the `_check_cromwell_status` method. Can you double check you're ok with that?

Reply:
Scratch that, I see why it's there now